plugins/vendor/zoom/lib/kZoomFlowManager.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @package plugins.Vendor
+ * @subpackage zoom
+ */
+
+class kZoomFlowManager implements kObjectCreatedEventConsumer
+{
+	const CONFIGURATION_PARAM_NAME = 'Zoom';
+	
+	
+	/**
+	 * @param BaseObject $object
+	 * @return bool
+	 */
+	public function objectCreated (BaseObject $object)
+	{
+		/* @var $object CaptionAsset */
+		$zoomConfiguration = kConf::get(self::CONFIGURATION_PARAM_NAME, kConfMapNames::MAP_NAME);
+		$object->setAccuracy($zoomConfiguration['ZoomTranscriptionAccuracy']);
```

Comment:
what happens if we dont have zoom map configured?
I would check $zoomConfiguration before trying to get property from it